<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bazaar::Internal::CloneOptionsPanel</class>
 <widget class="QWidget" name="Bazaar::Internal::CloneOptionsPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="optionsGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Options</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="useExistingDirCheckBox">
        <property name="toolTip">
         <string>By default, branch will fail if the target directory exists, but does not already have a control directory.
This flag will allow branch to proceed.</string>
        </property>
        <property name="text">
         <string>Use existing directory</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="stackedCheckBox">
        <property name="toolTip">
         <string>Create a stacked branch referring to the source branch.
The new branch will depend on the availability of the source branch for all operations.</string>
        </property>
        <property name="text">
         <string>Stacked</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="standAloneCheckBox">
        <property name="toolTip">
         <string>Do not use a shared repository, even if available.</string>
        </property>
        <property name="text">
         <string>Standalone</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="bindCheckBox">
        <property name="text">
         <string>Bind new branch to source location</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="switchCheckBox">
        <property name="toolTip">
         <string>Switch the checkout in the current directory to the new branch.</string>
        </property>
        <property name="text">
         <string>Switch checkout</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QCheckBox" name="hardlinkCheckBox">
        <property name="toolTip">
         <string>Hard-link working tree files where possible.</string>
        </property>
        <property name="text">
         <string>Hardlink</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QCheckBox" name="noTreeCheckBox">
        <property name="toolTip">
         <string>Create a branch without a working-tree.</string>
        </property>
        <property name="text">
         <string>No working-tree</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Revision:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="revisionLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
